lib/gui/pages/main/templates/main.tpl.html

```diff
@@ -95,8 +95,8 @@
         </progress-button>
 
         <p class="step-footer step-footer-split" ng-show="main.state.getFlashState().speed && main.state.getFlashState().percentage != 100">
-          <span>ETA: {{ main.state.getFlashState().eta | secondsToDate | amDateFormat:'m[m]ss[s]' }}</span>
           <span ng-bind="main.state.getFlashState().speed.toFixed(2) + ' MB/s'"></span>
+          <span>ETA: {{ main.state.getFlashState().eta | secondsToDate | amDateFormat:'m[m]ss[s]' }}</span>
```

Comment:
So is there some CSS-magic that ensures that the first span is left-aligned and the second span is right-aligned?